gnovm/pkg/gnolang/values.go

```diff
@@ -88,6 +88,14 @@ func (bv BigintValue) MarshalAmino() (string, error) {
 	return string(bz), nil
 }
 
+func (bv BigintValue) MarshalAminoJSON() (string, error) {
```

Comment:
I don't this is actually supported; but amino JSON should just be able to use the above MarshalAmino?